.travis.yml

```diff
@@ -1,17 +1,24 @@
 language: python
+
 python:
  - "2.6"
  - "2.7"
  - "3.3"
  - "3.4"
  - "3.5"
 
-# Twisted dropped Python 2.6 support, so install a compatible version on 2.6
 install:
+ # Twisted dropped Python 2.6 support, so install a compatible version on 2.6
  - pip install --upgrade pip setuptools
  - pip install -r test-requirements.txt
  - pip install -e .[commands,consumers]
+ - pip install codecov
```

Comment:
I've actually got this going in a separate PR (#417), but I like where your head is at :smile: